kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/README.md

```diff
@@ -67,20 +67,20 @@ The following parameters can be provided in the inputs file.
 | `adminServerName` | Name of the Administration Server. | `admin-server` |
 | `clusterName` | Name of the WebLogic cluster instance to generate for the domain. | `cluster-1` |
 | `configuredManagedServerCount` | Number of Managed Server instances to generate for the domain. | `2` |
-| `createDomainFilesDir` | Directory to get all the create domain scripts and supporting files, including the script that is specified by the createDomainScriptName property. By default, this directory is set to the relative path `wlst`, and the create script will use the built-in WLST offline scripts in `wlst` directory to create the WebLogic domain. It can also be set to the relative path `wdt`, and then the built-in WDT scripts will be used instead. An absolute path is also supported to point to an arbitrary directory in the file system. | wlst |
+| `createDomainFilesDir` | Directory to get all the create domain scripts and supporting files, including the script that is specified by the `createDomainScriptName` property. By default, this directory is set to the relative path `wlst`, and the create script will use the built-in WLST offline scripts in the`wlst` directory to create the WebLogic domain. It can also be set to the relative path `wdt`, and then the built-in WDT scripts will be used instead. An absolute path is also supported to point to an arbitrary directory in the file system. | `wlst` |
```

Comment:
in the`wlst` directory -> in the `wlst` directory.

Reply:
Got it, thanks.